.github/workflows/daily.yml

```diff
@@ -16,15 +16,14 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
 
   notifications:
-    # if: github.ref == 'refs/heads/master'
-    if: always()                              
+    if: always() && github.repository == 'fedimint/fedimint'
     name: "Notifications"
     timeout-minutes: 1
     runs-on: ubuntu-22.04
-    needs: [ check-unused-dependencies, audit ]
```

Comment:
We removed the `check-unused-dependencies` job in https://github.com/fedimint/fedimint/commit/b7a6e368192114376c1818f774f13dcde9803442.